server/meshmodel/kubernetes/v1.25.2/v1.0.0/relationships/k8s-resources-to-container.json

```diff
@@ -0,0 +1,102 @@
+{
+  "schemaVersion": "relationships.meshery.io/v1alpha3",
+  "version": "v1.0.0",
+  "kind": "hierarchical",
+  "type": "parent",
+  "subType": "alias",
+  "metadata": {
+    "description": "A hierarchical inventory relationship in which the configuration of (parent) component is patched with the configuration of other (child) component. Eg: The configuration of the EnvoyFilter (parent) component is patched with the configuration as received from WASMFilter (child) component."
+  },
+  "model": {
+    "schemaVersion": "models.meshery.io/v1beta1",
+    "version": "v1.0.0",
+    "name": "kubernetes",
+    "registrant": {
+      "kind": "artifacthub"
+    },
+    "model": {
+      "version": "v1.25.2"
+    },
+    "displayName": "Kubernetes",
+    "category": {
```

Comment:
Why it is necessary to include this information in a Relationship definition?

Reply:
...because the schema calls for a model $ref, which in order to fulfill to the letter of the json schema law means including the entire model definition. What I'm suggesting here is to:

1. Not honor the letter of the law, but the spirit of it with enough info for a model $ref (name, schemaVersion, kind, registrant,...)

Reply:
As discussed relationship schema needs to be revised.